testsuite/features/step_definitions/setup_steps.rb

```diff
@@ -0,0 +1,509 @@
+# Copyright (c) 2023 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+### This file contains all steps concerning setting up a test environment.
+### This may include the creation and handeling of SCC credentials, adding products, etc.
+
+require 'DateTime'
```

Comment:
The import makes sense to me since we are using it in the code below. I am wondering, though, why this is missing in `common_steps.rb`?
Did RubyMine suggested to add this?

Reply:
Good Question. I think VSCode yelled at me that a require was missing here. Interestingly it doesn't yell with `common_steps.rb` though

Reply:
I see that it is part of the `sdtlib` and the [official docs](https://ruby-doc.org/stdlib-2.5.9/libdoc/date/rdoc/DateTime.html) use another import for the Ruby version 2.5.9 we still use:

```ruby
require 'date'

DateTime.new(2001,2,3,4,5,6)
                    #=> #<DateTime: 2001-02-03T04:05:06+00:00 ...>
```

Reply:
Now a rubocop error I got makes sense. Is it still necessary to explicitly import it, when it's part of `stdlib` then?

Reply:
I do not think so. Because otherwise we would have it in `common_steps.rb` already.

Reply:
okay I will remove it :)